connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/ConnectProtocolCompatibilityTest.java

```diff
@@ -51,95 +41,58 @@ public class ConnectProtocolCompatibilityTest {
     private ConnectorTaskId taskId2x0 = new ConnectorTaskId(connectorId2, 0);
     private ConnectorTaskId taskId3x0 = new ConnectorTaskId(connectorId3, 0);
 
-    @Rule
-    public MockitoRule rule = MockitoJUnit.rule();
-
-    @Mock
-    private KafkaConfigBackingStore configStorage;
-    private ClusterConfigState configState;
-
-    @Before
-    public void setup() {
-        configStorage = mock(KafkaConfigBackingStore.class);
-        configState = new ClusterConfigState(
-                1L,
-                null,
-                Collections.singletonMap(connectorId1, 1),
-                Collections.singletonMap(connectorId1, new HashMap<>()),
-                Collections.singletonMap(connectorId1, TargetState.STARTED),
-                Collections.singletonMap(taskId1x0, new HashMap<>()),
-                Collections.emptySet());
-    }
-
-    @After
-    public void teardown() {
-        verifyNoMoreInteractions(configStorage);
-    }
```

Comment:
All of this is completely unnecessary and can be removed without diminishing testing coverage.